modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/Project/Overview/components/SubscriptionsOverview/components/AccountSubscriptionsList/components/AccountSubscriptionCard/AccountSubscriptionCard.jsx

```diff
@@ -20,16 +20,22 @@ import {
 	Skeleton,
 	StatusTag,
 } from '../../../../../../../../../../../common/components';
+import {useGetAccountSubscriptionUsage} from '../../../../../../../../../../../common/services/liferay/graphql/account-subscription-usage';
 import {
 	FORMAT_DATE_TYPES,
 	SLA_STATUS_TYPES,
 } from '../../../../../../../../../../../common/utils/constants';
+import {
+	productDisplayExepction,
+	subscriptionTypes,
+} from '../../../../../../../../../../../common/utils/constants/subscriptionCardsCount';
```

Comment:
PRODUCT_DISPLAY_EXEPCTION,
SUBSCRIPTION_TYPES,

```suggestion
import {
	PRODUCT_DISPLAY_EXEPCTION,
	SUBSCRIPTION_TYPES,
} from '../../../../../../../../../../../common/utils/constants/subscriptionCardsCount';
```

Reply:
I have a question for you guys, should be PRODUCT_DISPLAY_EXEPTION or PRODUCT_DISPLAY_EXCEPTION?